app/models/pay/subscription.rb

```diff
@@ -111,6 +113,10 @@ def active?
       ["trialing", "active"].include?(status) && (ends_at.nil? || on_grace_period? || on_trial?)
     end
 
+    def active_and_paused?
```

Comment:
```suggestion
    def active_but_paused?
```

That might give a better idea as to why it's "weird" to be checking both.

Reply:
I like it!